### DISCUSSION TITLE
How to use MMKV in Android native layer?

### DISCUSSION THREAD

**Discussion by vhow** [Tencent/MMKV#763] [Q&A]
How to achieve this?
1. Write key-value in Java layer
2. Read key-value in native (.cpp) layer

**Comment by lingol** [comment]
We do not officially support calling the native interface of MMKV on Android, because the native interface is **not part of public API**, and will be **changed** in the future **without further notice**.

**Comment by lingol** [comment]
It's officially supported in v2.1.0.
